kura/org.eclipse.kura.nm/src/main/java/org/freedesktop/modemmanager1/PropertyConnectionErrorStruct.java

```diff
@@ -0,0 +1,41 @@
+/*******************************************************************************
+ * Copyright (c) 2025 Eurotech and/or its affiliates and others
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *  Eurotech
+ *******************************************************************************/
+package org.freedesktop.modemmanager1;
+
+import org.freedesktop.dbus.Struct;
+import org.freedesktop.dbus.annotations.Position;
+
+/**
+ * Auto-generated class.
+ */
+public class PropertyConnectionErrorStruct extends Struct {
+
+    @Position(0)
+    private final String member0;
+    @Position(1)
+    private final String member1;
+
+    public PropertyConnectionErrorStruct(String member0, String member1) {
+        this.member0 = member0;
+        this.member1 = member1;
+    }
+
+    public String getMember0() {
+        return this.member0;
+    }
+
+    public String getMember1() {
+        return this.member1;
+    }
```

Comment:
As you might have noticed the autogenerated code is not the best. Here `member0` and `member1` is not really descriptive of the content of the struct. Looking at the documentation we can understand more:

<img width="1230" alt="image" src="https://github.com/user-attachments/assets/b38ac717-3564-43ba-8ab8-3a372658cc9a" />

In my original contribution, I left these as is since we wanted to limit the changes to the autogenerated code as much as possible. This allows us to update the autogenerated code almost painlessly at the cost of some readability.

Let's keep them as `memberX` for consistency but be aware of the trade-off we're making.